test/fixtures/sanitation/plain-text-response-body.js

```diff
@@ -12,6 +12,8 @@ hooks.after('Resource > Update Resource', (transaction, done) => {
   transaction.test.expected.body = body.replace(tokenPattern, '--- CENSORED ---');
 
   // Sanitation of diff in the patch format
-  delete transaction.test.results.body.results.rawData;
+  // This is some dark magic, as "gavelResult.fields[name].errors" is a Error[].
+  // It must never contain "rawData" property. Somehow, this works.
+  delete transaction.test.results.body.errors.rawData;
```

Comment:
I'd be interested in why this works 🤔 Could it be something we are missing in what Gavel produces? Or a magical behavior of `delete`?

Reply:
Runing
```js
const foo = { a: [] }
delete foo.a.b
// true
```
returns a boolean and does nothing to the actual object. I believe it should be safe to remove this line. Let's see what the tests have to say.

Reply:
I think what the line is trying to achieve is it is trying to remove the diff data from the validation result, for sanitation purposes. So it shouldn't be removed, but replaced with a line which truly does remove the diff.

The test probably isn't thorough enough to detect that the diff data in fact weren't removed 😬